test/e2e/common/setup-kubestellar.sh

```diff
@@ -142,6 +155,15 @@ kflex ctx --overwrite-existing-context its1
 
 kflex ctx
 
+# Check required kubeconfig contexts exist after they are created
+REQUIRED_CONTEXTS=("$HOSTING_CONTEXT" "its1" "wds1")
+for ctx in "${REQUIRED_CONTEXTS[@]}"; do
+    if ! kubectl config get-contexts "$ctx" >/dev/null 2>&1; then
+        echo "ERROR: kubeconfig context '$ctx' does not exist. Exiting." >&2
+        exit 1
+    fi
+done
+
```

Comment:
We do not need this. It is clear enough from the existing diagnostics what contexts were missing --- error messages called it out.